### DISCUSSION TITLE
LockScreen image instead of (or additional to) wallpaper

### DISCUSSION THREAD

**Discussion by xcassanello** [jac0b-w/album-art-wallpaper#42] [Ideas]
The idea is pretty simple. Add support to change also the lockscreen.

Superb option would be to add in the UI an option to change Wallpaper, lockscreen or both.

By the way, any issue with installer on Windows 10 is solved by disabling SmartScreen.

**Comment by jac0-b** [comment]
Yes I would love to add this feature. Windows is much more awkward for changing the lock screen (who knows why) but it should be doable.

Also you should be able to allow unknown installers on a per app basis without actually disabling any windows defender features.
